### DISCUSSION TITLE
Keep kitty terminal visible when opening a gui app?

### DISCUSSION THREAD

**Discussion by sw1nn** [kovidgoyal/kitty#6840] [Q&A] [answered]
When I open a gui app from kitty the kitty terminal is 'replaced' by the gui app. When the app closes kitty reappears. 

Is there a configuration option to stop this happening (i.e. leave the kitty window visible). I wonder if this is a misunderstanding on my part about how kitty is 'supposed' to behave?

e.g. running in wayland I might want to run `wev` to debug key presses. I would like to open the `wev` window, press some keys, and see the log output in the original terminal, but when I type `wev` at the terminal. the `wev` checkerboard window opens in place of kitty and the log output is not available until I close `wev`

system is arch linux running Hyprland wayland compositor

**Comment by kovidgoyal** [comment, marked as answer]
That is up to your window manager, not kitty.

**Reply by sw1nn** [reply]
🤦🏻‍♂️ Thanks for this. 

My confusion was because it was only happening for kitty and not other terminals. It turns out I had this:
```
    enable_swallow = true
    swallow_regex = ^(kitty)$
```
in my WM config.

Sorry for the noise.
